### DISCUSSION TITLE
pressure out of range

### DISCUSSION THREAD

**Discussion by lvdbrand** [idaholab/moose#26297] [Q&A Modules: Porous Flow]
Hello,

I'm trying to make a 2D terrain stability study by coupling the "TensorMechanics" module and the "PorousFlow" module. 
I'd like to simulate a static water table in the ground. To do this, I've used an initial condition to give the "porepressure" variable a value using the "PieceWiseMultilinear" function.  My mesh is divided into two blocks: only the lower block contains water. At the interface between these two blocks ("aquifers" boundary), the porepressure variable is set to 0. 
My problem is that Moose returns this error : 
![image](https://github.com/idaholab/moose/assets/78745794/a5653ee1-6b7f-44f6-8fae-31c06694139e)

Why the pressure is out of range? Is there a lower limit below which Moose will not take a pressure into account? If so, how can this problem be solved? 

Thanks!

Lucas

Here is my code with my mesh and the file with the values of the "porepressure" variable:
[pressure_out_of_range.zip](https://github.com/idaholab/moose/files/13638333/pressure_out_of_range.zip)

**Comment by GiudGiud** [comment]
Hello

The lower limit on pressure is from the water fluid properties object, which requires physical pressures

The final Moose error messages is simply from enough time steps failing. It's not the one to look at. You want to address the problem with the pressures

Guillaume

**Reply by lvdbrand** [reply]
The problem is with "Water97FluidProperties"? Should I use something else like "SimpleFluidProperties" in fluidproperties object?

**Reply by GiudGiud** [reply]
Feel free to use that one to get a result, but if you want to use water properties it would be better to find out why these properties are being evaluated at such a terribly small pressure

**Reply by lvdbrand** [reply]
To mimic an aquifer, I create a hydrostatic pressure whose value starts at 0 at the "aquifers" boundary and increases towards the "bottom" boundary. That's why the pressure is so small. Is there an easier way of doing this?

Thanks

**Reply by GiudGiud** [reply]
Can you offset pressure to have realistic values? I reckon 0 atm does not exist in the real physical system

**Reply by 1runer** [reply]
Hydrostatic pressure is not "0". You always have the atmospheric pressure acting on the fluid and rock body which is in the order of 1e5 Pa.
